book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
```

Comment:
- [ ] To implement the `has-many` side of the relationship(a book has many rentals), a method for adding rentals should be present. Something like so: 

```ruby
  def add_rental(person, date)
    Rental.new(date, self, person)
  end
```
Checkout this [lesson](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship) and kindly fix that.